runtime-builder/go-build.sh

```diff
@@ -44,8 +44,9 @@ mv "${staging}" "${workspace}"/app
 
 # Generate application Dockerfile.
 cat > Dockerfile <<EOF
-FROM gcr.io/google-appengine/debian8@${DEBIAN_DIGEST}
+FROM gcr.io/distroless/base:latest
```

Comment:
Don't use "latest".  Use the same technique that we did for grabbing debian8's latest digest tag.